website/content/docs/connect/proxies/integrate.mdx

```diff
@@ -201,35 +159,29 @@ the PreparedQuery API does not support blocking, so proxies choosing to
 populate endpoints in memory will need to poll the endpoint at a suitable and
 ideally configurable frequency.
```

Comment:
GH won't let me comment on the entire prepared query section since this wasn't modified in the PR, but I find this to be lacking details. The following would be more descriptive IMO.

> Upstreams may be defined with the Prepared Query target type. These upstreams
should use Consul's [prepared query](/api/query#execute-prepared-query) API to determine a list of upstream endpoints for the service. It's worth noting that
the PreparedQuery API does not support blocking, so proxies choosing to
populate endpoints in memory will need to poll the endpoint at a suitable and
ideally configurable frequency.